### PR TITLE
Bookstore: initialize project with components

### PR DESCRIPTION
**Pull request by codedit334**
## Hi Reviewer, 👋 

In this PR:
- Setup a new React application using [Create React App](https://reactjs.org/docs/create-a-new-react-app.html#create-react-app).
- Created a directory for your reusable components: /components.
- Installed [React Router](https://reactrouter.com/en/main/start/tutorial#setup).
- [Setup your router](https://reactrouter.com/en/main/start/tutorial#adding-a-router) inside of <App>.
- Add 2 routes in your application:
  - `/`, which contains:
    - A book list, where each book has a title and author.
    - A form that allows for creating a new book.
  -  `/categories`:
     - This should display a button that checks for a status.
- Include a navigation component that includes links to the 2 routes you created.

# 🤝

### CODE REVIEW

File: src/components/Books.js
Comment (christianbiring1) [inline src/components/Books.js:1]: - [ ] As mentioned in the project requirements, Kindly make sure you have a component called `Book` for displaying a single book means it receives a title and an author as prop from its parent `BookList` in this way you will have your UI divided into small chunks of reusable components which is the whole concept behind `react`.

```jsx
const BookList() => {
// have arrays of books
return(
<div>
// map the arr of books.
<Book author={book.author} title={book.title}
</div>
)
}
// Then the bookList will be imported into the Books components.
```

Comment (christianbiring1) [reply src/components/Books.js:1]: - [ ] also the form to add a book should be a separate component.

YOu will have to have
```jsx
const NewBookForm = () => {
// The book form goes here and then the component is exported in the Books Components.
}
```

File: src/components/Books.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+            </span>
+            <br />
+            <span className="book-author">
+              Author:
+              {book.author}
+            </span>
+            <br />
+            <hr />
+            <br />
+          </li>
```
Comment (christianbiring1) [inline src/components/Books.js]: - [ ] Kindly make sure you also have a button to delete(no functionality required yet) a book on each created book :smile:  That way the user will be able to delete a book in the future.

**Review by christianbiring1 — CHANGES_REQUESTED** [review]
Hi @codedit334 :wave: , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
# STATUS: REQUEST CHANGES :recycle: 
soclose

## Highlights
- [ ] All linters checks are green :heavy_check_mark: 
- [ ] Descriptive PR `title` and `summary` :+1: 
- [ ] The `Router` are set u correctly

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by m05e5 — APPROVED** [review]
**Hi @codedit334 
You did great. Your project is complete!  There is nothing else to say other than... it's time to merge:shipit: 
Congratulations! 🎉 
CleverScornfulBuffalo-size_restricted

## Highlights


✔️ Clean Codebase
✔️ Great job implementing the requested changes

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

_**m05e5**_

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._**

**Pull request merged**